examples/js/ThreeLoader.js

```diff
@@ -0,0 +1,78 @@
+/* global itowns, Promise */
+var manager = new itowns.THREE.LoadingManager();
+
+// namespace ThreeLoader.
+var ThreeLoader = {};
+
+// Utility method to use Promise.
+function defer() {
+    var deferedPromise = {};
+    deferedPromise.promise = new Promise(function _(resolve, reject) {
+        deferedPromise.resolve = resolve;
+        deferedPromise.reject = reject;
+    });
+    return deferedPromise;
+}
+
+/**
+ * Load a script asynchronously
+ * @param {String} uri URI of the script to load.
+ * @returns {Promise} A promise resolved when the script is loaded.
+ */
+function loadScriptAsync(uri) {
+    var deferedPromise = defer();
+    var firstScriptTag = document.getElementsByTagName('script')[0];
+    var tag = document.createElement('script');
+    tag.async = true;
+    tag.onload = function r() {
+        deferedPromise.resolve();
+    };
+    tag.src = uri;
+    firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);
```

Comment:
No need for that imho, use something like `document.body.append(tag)` instead.

Reply:
ok, good idead